pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md

```diff
@@ -0,0 +1,444 @@
+? Choose your default database client 
+  sqlite 
+❯ postgres 
+  mysql
+</code></pre>
+
+Then, enter your PostgreSQL database parameters, and select No when prompted for enabling SSL connection (We will review that point later)
+
+<pre class="console"><code>
+? Choose your installation type Custom (manual settings)
+? Choose your default database client postgres
+? Database name: defaultdb
+? Host: postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net
+? Port: 20184
+? Username: avnadmin
+? Password: ********************
+? Enable SSL connection: No
+
+Creating a project with custom database options.
+Creating a new Strapi application at /home/my/app/path/my-strapi.
+Creating files.
+Dependencies installed successfully.
+
+Your application was created at /home/my/app/path/my-strapi.
+
+Available commands in your project:
+
+  yarn develop
+  Start Strapi in watch mode. (Changes in Strapi project files will trigger a server restart)
+
+  yarn start
+  Start Strapi without watch mode.
+
+  yarn build
+  Build Strapi admin panel.
+
+  yarn strapi
+  Display all available commands.
+
+You can start by doing:
+
+  cd /home/my/app/path/my-strapi
+  yarn develop
+
+Done in 662.54s.
+</code></pre>
+
+You could now start you strapi application with yarn commands as described above, but we need before to modify a SSL parameter.
+If you don't, this is what happen :
+
+<pre class="console"><code>$ yarn develop
+yarn run v1.22.17
+warning ../../../package.json: No license field
+$ strapi develop
+Building your admin UI with development configuration ...
+Admin UI built successfully
+[2021-12-31 13:16:32.632] debug: ⛔️ Server wasn't able to start properly.
+[2021-12-31 13:16:32.633] error: no pg_hba.conf entry for host "109.190.xxx.xxx", user "avnadmin", database "defaultdb", no encryption
+error: no pg_hba.conf entry for host "109.190.xxxx.xxxx", user "avnadmin", database "defaultdb", no encryption
+    at Parser.parseErrorMessage (/home/my/app/path/my-strapi/node_modules/pg-protocol/dist/parser.js:287:98)
+    at Parser.handlePacket (/home/my/app/path/my-strapi/node_modules/pg-protocol/dist/parser.js:126:29)
+    at Parser.parse (/home/my/app/path/my-strapi/node_modules/pg-protocol/dist/parser.js:39:38)
+    at Socket.<anonymous> (/home/my/app/path/my-strapi/node_modules/pg-protocol/dist/index.js:11:42)
+    at Socket.emit (node:events:390:28)
+    at Socket.emit (node:domain:475:12)
+    at addChunk (node:internal/streams/readable:315:12)
+    at readableAddChunk (node:internal/streams/readable:289:9)
+    at Socket.Readable.push (node:internal/streams/readable:228:10)
+    at TCP.onStreamRead (node:internal/stream_base_commons:199:23)
+error Command failed with exit code 1.
+info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
+</code></pre>
+
+This is a known bug, you can follow the issue here: https://github.com/strapi/strapi/issues/12058.
+
+Whe fix it by manually modifying the database configuration file.
```

Comment:
"We can fix it by manually modifying the database configuration file."

Reply:
Done